release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -98,6 +98,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-nodes"]
 === Nodes
 
+[id="ocp-4-12-node-cgroups-v2"]
+==== Linux control groups version 2 promoted to Techology Preview
+
+{product-title} support for link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) has been promoted to Technology preview. Cgroups v2 is the next version of the kernel link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01[control groups] and offers multiple improvements.
```

Comment:
From a review of the OCP docs library, I see that cgroup is stated in several ways:

* cgroup
* cgroups
* Cgroup
* Cgroups
* CGroup
* CGroups

In https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html, the following is mentioned:

> “cgroup” stands for “control group” and is never capitalized.

I also noticed singular or plural use of the term in that document. 

Given this, you could consider changing "Cgroups" to "cgroups" and using the lowercase version of the term throughout the PR.

Reply:
You could consider updating "and offers multiple improvements" to something like the following examples:

> and v2 offers multiple improvements
> and it offers multiple improvements
> and the update offers multiple improvements

Alternatively, you could split that sentence into two and provide some example high-level improvements. For example:

```
cgroups v2 is the next version of the kernel [control groups](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01). cgroups v2 offers multiple improvements, including <improvements>.
```

Reply:
@pneedle-rh Thank you for your review. You are correct about your two typo comments. I did catch those on my final review but failed to push those changes. Sorry to have wasted your time. I am taking your third suggestion, to split the sentence. Sounds much better. Thanks again.

Reply:
Thank you Michael. I noticed that for some reason the link in my example uses Markdown format. I mean to use AsciiDoc format:

```
cgroups v2 is the next version of the kernel link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01[control groups]. cgroups v2 offers multiple improvements, including <improvements>.
```